src/CLI/CLI.cs

```diff
@@ -9,45 +9,41 @@ namespace CppSharp
 {
     class CLI
     {
-        private static Options _options = new Options();
+        private static Mono.Options.OptionSet CLIOptions = new Mono.Options.OptionSet();
```

Comment:
Please rename to "optionSet". Fields should not start with a capital letter and it's also redundant to use a prefix named after the parent class.